tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/Net/ValidateAsyncIntegrationV5AndV7.cs

```diff
@@ -18,11 +18,11 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.GraphQL.Net
         ReturnTypeName = ClrNames.GenericParameterTask,
         ParameterTypeNames = new[] { "GraphQL.Validation.ValidationContext", "System.Collections.Generic.IEnumerable`1[GraphQL.Validation.IValidationRule]" },
         MinimumVersion = GraphQLCommon.Major5,
-        MaximumVersion = GraphQLCommon.Major5,
+        MaximumVersion = GraphQLCommon.Major7,
```

Comment:
Would it make sense to duplicate this InstrumentMethod where one has `MinimumVersion=MaximumVersion=GraphQLCommon.Major5` and the other has `MinimumVersion=MaximumVersion=GraphQLCommon.Major7`? That could we safer so we never include v6 in the version range. I believe there are some examples in this repo already where we've done this but with different assembly names

Reply:
> As far as I know, version 6 never existed. There is no release of version 6 in the repo of graphql-dotnet nor in nuget

Correct, they skipped v6. I guess there's an argument they could add one, but doesn't seem critical